docs/migrating/pgbouncer.md

```diff
@@ -40,7 +40,7 @@ select
   usename,
   application_name
 from pg_stat_activity
-where application_name ilike '%Supavisor%'
+where application_name in ('Supavisor', 'Supavisor auth_query')
```

Comment:
@chasers including the auth query one.

Reply:
But why? Now we have to remember to add any here if we update them.